libs/shared/src/consts/providers/provider-credentials.ts

```diff
@@ -5,7 +5,7 @@ const mailConfigBase: IConfigCredentials[] = [
   {
     key: CredentialsKeyEnum.From,
     displayName: 'From email address',
-    description: 'Use the authenticated email address from the delivery provider you will send emails from.',
+    description: 'Use the same email address you used to login to your email provider',
```

Comment:
As I remember it the authenticated email is the one the user needs to provide there (due to the reason you can add their multiply email but the authenticated email only one).
Do you think the user will provide the relevant one if we do the change here?

Reply:
I agree here with @djabarovgeorge it's not the same one that is used for login. It's the authenticated email in the SMTP server of the delivery provider. They are not automatically connected. @oba2311

Reply:
@scopsy gotcha, it wasn't clear in the original sentence English a bit fuzzy - how about 
`Use the same email address you used to authenticate your email provider`

This is clear English, but I'd assume people wouldn't remember which address they used to authenticate...

Reply:
They probebly should 
GendGrid for example

Reply:
They should remember it, the correct email in order to use Sendgrid for example 

![image](https://user-images.githubusercontent.com/39195835/184889890-57b7a285-298b-4ba9-bbfb-c571b2d98ecc.png)